int64 id
string name
float64 present_position
float64 goal
float64 present_error
float64 present_speed
float64 present_load
float64 present_voltage
float64 present_temperature
bool compliant
float64 max_angle
float64 min_angle
bool direction
string model
float64 offset
float64 max_load

